src/main/java/com/inqoo/trainingservice/app/job/JobService.java

```diff
@@ -19,15 +19,15 @@
 
 @Service
 public class JobService {
-    private final UnavailabilityRepository unavailabilityRepository;
+    private final AbsenceRepository absenceRepository;
     private final JobRepository jobRepository;
     private final OfferRepository offerRepository;
     private final TrainerRepository trainerRepository;
 
-    public JobService(UnavailabilityRepository unavailabilityRepository,
+    public JobService(AbsenceRepository unavailabilityRepository,
```

Comment:
Instead of using repository directly we can create a projection of absence and use it here.